packit/base_git.py

```diff
@@ -74,19 +87,6 @@ def specfile_path(self) -> str:
 
         return self._specfile_path
 
-    @property
-    def specfile(self) -> SpecFile:
-        # changing API is fun, where else could we use inspect?
-        s = inspect.signature(SpecFile)
-        if "changelog_entry" in s.parameters:
-            return SpecFile(
-                path=self.specfile_path,
-                sources_location=self.specfile_dir,
-                changelog_entry=None,
-            )
-        else:
-            return SpecFile(path=self.specfile_path, sources_location=self.specfile_dir)
-
```

Comment:
I'm not getting the difference but If you like than why not...

Reply:
I'll change it, I'm assuming this is an outcome of the rebase.

Reply:
No problem with that, I was only trying to find the logic behind...;-)